app/routes/admin/users.js

```diff
@@ -3,4 +3,59 @@ import Ember from 'ember';
 const { Route } = Ember;
 
 export default Route.extend({
+  titleToken() {
+    return this.i18n.t('Users');
+  },
+  model() {
+    return [{
+      name        : 'Testname 1',
+      email       : 'Testname1@gmail.com',
+      status      : 'active',
+      systemRoles : 'unverified user',
+      eventRoles  : 'role1',
+      userLinks   : ['Sessions', 'Events', 'Tickets', 'Settings'],
```

Comment:
These user links are constant for all users. Please actually try out the current system to see how things are.